tutorial-02: updates response examples (#60)
inveniosoftware/training: 07-data-models-new-field/README.md

{"status": 400, "message": "Validation error.", "errors": [{"field": "owner", "message": "Not a valid integer."}]}
```

It seems that our request wasn't successful. By checking again the error message we can see that in our request
the `owner` field has a `string` value rather than an `integer`. But who validated that?

> If you remember we talked earlier about `loaders` and specifically we updated our `marshmallow` schema. But how is it related? To answer that let's talk about what is the responsibility of the `loaders`. It's purpose is to load the data which is passed by a request to create a new record, validate it by using our `marshmallow` schema and transform it into our internal representation.

Reviewer: ```suggestion
If you remember we talked earlier about `loaders` and specifically we updated our `marshmallow` schema. But how is it related? To answer that let's talk about what is the responsibility of the `loaders`. Its purpose is to load the data which is passed from the request to create a new record, validate it using our `marshmallow` schema and transform it into our internal representation.
```

Reviewer 2: minor:
```diff
- Its purpose is to load the data which is passed from the request to create a new record ...
+ Its purpose is to load the data received in a request to create a new record ...
```

```json
{
  "_bucket": "60f2b083-8f7b-4aba-a00e-09e3bb3e12af",
  "created": "2019-11-25T07:41:44.620275+00:00",
  "id": "1",
  "links": {
    "files": "https://localhost:5000/api/records/1/files",
    "self": "https://localhost:5000/api/records/1"
  },
  "metadata": {

>     "$schema": "https://my-site.com/schemas/records/record-v1.0.0.json",

Reviewer: moderate: Is not the fault of the work, maybe Invenio is not fully working. Is this supposed to be in the output?

Reviewer 3: This was changed, in my pr #71
